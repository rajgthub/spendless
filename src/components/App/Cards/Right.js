import React from 'react'
import SpendLessFilter from '../SpendLessFilter/index';
 const Right = () => (
     <div className="card_right">
     <SpendLessFilter />   
     </div>
 )
export default Right
