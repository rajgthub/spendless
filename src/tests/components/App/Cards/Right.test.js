import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Right from '../../../../components/App/Cards/Right';
test("should render Header ", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Right/>);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
