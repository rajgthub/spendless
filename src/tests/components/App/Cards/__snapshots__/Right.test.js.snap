// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header  1`] = `
<div
  className="card_right"
>
  <Connect(SpendLessFilter) />
</div>
`;
